v2.5/getting-started/kubernetes/installation/hosted/kubernetes-datastore/index.md

```diff
@@ -65,21 +66,14 @@ networking with a full node-to-node mesh and/or explicit configuration of peers.
 To install Calico with Calico networking, run one of the commands below based on your Kubernetes version.
 This will install Calico and will initially create a full node-to-node mesh.
 
-For **Kubernetes 1.6+** clusters:
+> Note: Calico `v2.5.0` or higher only support Kubernetes `v1.7.0` or higher.
```

Comment:
typo: `supports`

I also think the phrasing could be improved a little:

Note: Calico v2.5 with Kubernetes-Backend requires Kubernetes v1.7+.

?